src/devices/wholeBodyDynamics/WholeBodyDynamicsDevice.h

```diff
@@ -347,6 +392,25 @@ class WholeBodyDynamicsDevice :  public yarp::dev::DeviceDriver,
 
     /** F/T sensors interfaces */
     std::vector<yarp::dev::IAnalogSensor * > ftSensors;
+    //std::vector<yarp::dev::ISixAxisForceTorqueSensors * > ftSensors;
+
+    /** Remapped multiple analog sensors containing the sensors that implement multiple analgo sensor interfaces*/
+    yarp::dev::PolyDriver multipleAnalogRemappedDevice;
+    struct
+    {
+        yarp::dev::ITemperatureSensors * temperatureSensors;
+        yarp::dev::ISixAxisForceTorqueSensors * ftMultiSensors;
+        yarp::dev::IMultipleWrapper * multwrap;
+        //yarp::dev::IThreeAxisLinearAccelerometers * linearAccelerometers;
+//        yarp::dev::IOrientationSensors * orientationSensors;
+//        yarp::dev::IThreeAxisGyroscopes * gyroscopes;
+//        yarp::dev::IThreeAxisMagnetometers * magnetometers;
```

Comment:
Can we remove this?

Reply:
I was thinking about leaving it in case someone wanted to extend it. Although from the current implementation of embObjFT and embObjfIMU they seem to be meant to be used always separated.  So we could.

Reply:
Removed